dds_web/forms.py

```diff
@@ -68,7 +68,7 @@ class RegistrationForm(flask_wtf.FlaskForm):
     )
 
     policy_checkbox = wtforms.BooleanField(
-        "I agree to the conditions",
+        "I have read and agree to the Privacy Policy and User Agreement documents linked above.",
```

Comment:
```suggestion
        "I have read and I agree to the Privacy Policy and User Agreement documents linked above.",
```
but I am not at all sure which one is correct/better. Maybe Aishling can help here

Reply:
IMO either is clear so would stick with the existing one.